frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -93,6 +103,23 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({ vm, vmi, pods, mi
           DASH
         )}
       </dd>
+      <dt>
+        CD-ROMs
+        {canUpdateVM && (
```

Comment:
To stay consistent, the button should go to the value instead of the section description.
Please have a look at `EditButton` component within https://github.com/openshift/console/pull/2392

Reply:
By design, the Edit button should be by the section title. I switched to `EditButton`